src/handlebar-helpers.js

```diff
@@ -0,0 +1,34 @@
+// list of useful handlebars helpers we can bundle
+// credit goes to https://stackoverflow.com/a/31632215 & https://stackoverflow.com/a/50427569
+//
+// TODO: maybe make this extensible via another CR field?
+
+const objectPath = require('object-path');
+
+const helpers = {
+  assign: function (varName, varValue, options) {
+    if (!isNumber(varName)) {
+      objectPath.set(options, ['data', 'root', varName], varValue);
+    } else {
+      throw Error(`Cannot assign number: ${varName}`); 
+    }
```

Comment:
```suggestion
    if (!objectPath.has(options, 'data.root') {
      objectPath.set(options, 'data.root', {});
    }
    objectPath.set(options, ['data', 'root', varName], varValue);
```

Reply:
numbers are ok, as long as we ensure root has already been defined as an object

Reply:
This will prevent it from being transformed into an array?

Reply:
correct

Reply:
oh I misunderstood you